test/test_indexing.py

```diff
@@ -1110,10 +1110,10 @@ def test_getitem_scalars(self, device):
         self.assertEqual(a[0, 1], a[zero, one])
         self.assertEqual(a[0, one], a[zero, 1])
 
-        # indexing by a scalar should slice (not copy)
-        self.assertEqual(a[0, 1].data_ptr(), a[zero, one].data_ptr())
-        self.assertEqual(a[1].data_ptr(), a[one.int()].data_ptr())
-        self.assertEqual(a[1].data_ptr(), a[one.short()].data_ptr())
+        # indexing by a scalar should copy
+        self.assertNotEqual(a[0, 1].data_ptr(), a[zero, one].data_ptr())
+        self.assertNotEqual(a[1].data_ptr(), a[one.int()].data_ptr())
+        self.assertNotEqual(a[1].data_ptr(), a[one.short()].data_ptr())
```

Comment:
@Chillee do you need this PR to go through right now? I traced the blame of this back to https://github.com/pytorch/pytorch/pull/6426 and #6217, so I think it would be good if we could run it by Sam (but he's on PTO).

Reply:
Haha that's funny/interesting.